client/core/core_test.go

```diff
@@ -2689,7 +2701,8 @@ func TestSend(t *testing.T) {
 	}
 }
 
-func TestTrade(t *testing.T) {
+func trade(t *testing.T, async bool) {
+	t.Helper()
```

Comment:
Isn't `t.Helper()` actually _undesirable_ in this use case?  The calling function has just one line, so we are interested in the line number in this function that failed, not the caller.

Reply:
oh, yeah.